src/prpl.ts

```diff
@@ -164,6 +165,7 @@ export function makeHandler(root?: string, config?: Config): (
       "https://*.adjust.com",
       "maps.googleapis.com",
       "*.adform.net",
+      "http://212.98.242.24:8561",
```

Comment:
abi domainleri yok mu?

Reply:
Abi selamlar, o test içinmiş, prod ve stage için kullanılan domain ile değiştirdim.